treeherder/perf/migrations/0021_remove_application_from_constraint.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.6 on 2019-11-11 16:25
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0015_add_repository_tc_root_url'),
+        ('perf', '0020_add_application_field'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='performancesignature',
+            unique_together={('repository', 'framework', 'signature_hash'), ('repository', 'suite', 'test', 'framework', 'platform', 'option_collection', 'extra_options', 'last_updated')},
```

Comment:
This line does match https://github.com/mozilla/treeherder/pull/5625/files#diff-a6cd038d57ff9d2bf9dd3e06ec80d7fdR93  It seems to reverse `0020`, I would assume we can find a previous migration that has exactly this line.

Reply:
This migration will perform an `ALTER TABLE` on the database, I assume it is the signatures table, so it should be fast.

Reply:
I would be nice to have the generated SQL, so it can be run on a sample database.